contracts/lib/OrderFulfiller.sol

```diff
@@ -157,10 +157,13 @@ contract OrderFulfiller is
         bytes32 fulfillerConduitKey,
         address recipient
     ) internal {
-        // Derive order duration, time elapsed, and time remaining.
-        uint256 duration = orderParameters.endTime - orderParameters.startTime;
-        uint256 elapsed = block.timestamp - orderParameters.startTime;
-        uint256 remaining = duration - elapsed;
+        // Skip underflow checks as we know orderParameters.startTime <= block.timestamp < orderParameters.endTime
```

Comment:
Two comments:
- break comment up so that 80 character line limit is not exceeded
- avoid "we" in favor of declarative comments (basically remove "we know")